buildSrc/build.gradle.kts

```diff
@@ -69,3 +72,30 @@ configure<VersionCatalogUpdateExtension> {
     keepUnusedPlugins.set(true)
   }
 }
+
+apply<SpotlessPlugin>()
+configure<SpotlessExtension> {
+  kotlin {
+    ktlint().editorConfigOverride(
+      mapOf(
+        "disabled_rules" to "no-wildcard-imports,filename",
+        "indent_size" to 2
+      )
+    )
+    target("**/*.kt")
+  }
+
+  kotlinGradle {
+    ktlint().editorConfigOverride(
+      mapOf(
+        "disabled_rules" to "no-wildcard-imports",
+        "indent_size" to 2
+      )
+    )
+    target("**/*.gradle.kts")
+  }
+}
+
+val isCI by extra(System.getenv("CI") == "true")
+
+if (!isCI) { tasks.withType<KotlinCompile> { dependsOn("spotlessKotlinApply", "spotlessKotlinGradleApply") } }
```

Comment:
Actually, in the CI, if any of the files in buildSrc/ is ill-formated, will `spotlessCheck` task fail?

Reply:
Yes, spotlessCheck fails when file is ill-formatted and CI environment variable is set.